redundant-bmc/src/rbmc_tool.cpp

```diff
@@ -117,6 +136,11 @@ sdbusplus::async::task<> displayLocalBMCInfo(sdbusplus::async::context& ctx,
             {
                 printNoRedReasons();
             }
+
+            if ((role == "Active") && enabled && paused)
```

Comment:
I believe we need to print the FO pause reason in the Passive BMC as well, but I don't think we've decided yet how it will obtain the FO pause reason.

Reply:
Yea, I think we'll want to do that when we have the data on that BMC.  I have stories opened to deal with the passive side of things.